xml/System.Reflection.Emit/AssemblyBuilder.xml

```diff
@@ -76,7 +76,18 @@
   </Attributes>
   <Docs>
     <summary>Defines and represents a dynamic assembly.</summary>
-    <remarks>For more information about this API, see <see href="/dotnet/fundamentals/runtime-libraries/system-reflection-emit-assemblybuilder">Supplemental API remarks for AssemblyBuilder</see>.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+For more information about this API, see [Supplemental API remarks for AssemblyBuilder](/dotnet/fundamentals/runtime-libraries/system-reflection-emit-assemblybuilder).
+
+> [!WARNING]
```

Comment:
Wonder if it should be `Warning` or `Note` or ...

Reply:
I think Warning is good enough as it shows up in the preview: https://review.learn.microsoft.com/en-us/dotnet/api/system.reflection.emit.persistedassemblybuilder?view=net-9.0&branch=pr-en-us-10541

Reply:
Yeah saw that, still not sure Warning   
<img width="656" alt="image" src="https://github.com/user-attachments/assets/d12bf159-2eb6-4160-904f-39ab15e44f89">
or Note
<img width="658" alt="image" src="https://github.com/user-attachments/assets/8b6d904f-182f-42ad-92d4-d49288be139b">

Reply:
I think warning is appropriate.

Reply:
OK, warning then